code/datums/outfits/outfit_admin.dm

```diff
@@ -256,6 +256,11 @@
 	if(visualsOnly)
 		return
 
+	var/obj/item/weapon/rig/light/ninja/rig = new /obj/item/weapon/rig/light/ninja(src)
+	if(H.dna)
+		rig.dnaLock = H.dna
```

Comment:
You can just do `rig.dnaLock = H.dna`. If the latter is null, then the former will just be null as well.